bin/sentieon_init.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
I suspect that there is something with the sentieon-license here that I haven't fully understood yet :-/  When is this "localhost license file" relevant? I made a couple of Sentieon-nf-modules over at nf-core/modules:

https://github.com/nf-core/modules/tree/master/modules/nf-core/sentieon/bwamem

https://github.com/nf-core/modules/blob/master/modules/nf-core/sentieon/setup.md

I've tested the sentieon-nf-modules with the test-license provided to nf-core by Don Freed of Sentieon and with DNGC's real commercial license for sentieon. In both cases `SENTIEON_LICENSE_BASE64` is a base64-incoded url.

 I see you are testing whether `LICENSE_ENCODED` (which is `SENTIEON_LICENSE_BASE64`) is of length more than 1500 characters. When would it ever be such a long string?